pysite/views/wiki/history/compare.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+import difflib
+
+from flask import abort
+from pygments import highlight
+from pygments.formatters import HtmlFormatter
+from pygments.lexers import DiffLexer
+
+from pysite.base_route import RouteView
+from pysite.mixins import DBMixin
+
+
+class CompareView(RouteView, DBMixin):
+    path = "/history/compare/<string:first_rev>/<string:second_rev>"
+    name = "history.compare"
+
+    table_name = "revisions"
+    table_primary_key = "id"
+
+    def get(self, first_rev, second_rev):
+        before = self.db.get(self.table_name, first_rev)
+        after = self.db.get(self.table_name, second_rev)
```

Comment:
Because of the way you implemented the template, you will likely need to check which one comes first, and also whether they're equal.

Reply:
Fixed in latest commit.